### PR TITLE
feat: Alert Component Added

### PR DESCRIPTION
**Pull request by ShreyashMahajan**
Alert components have been added.
There are four types of alerts are added that are info, success, warning, and error.
Along with that Alert composition is added. 
Please review the code and help for the improvement

### CODE REVIEW

**Comment by Namrata-J** [comment]
the code is clean and understandable, looks good to me, you can merge it!!

**Review by ishapatell1 — APPROVED** [review]
Looks good to merge.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5502
+}
```
Comment (ultronjs) [inline .vscode/settings.json:3]: You can put these type of files in gitignore

**Review by ultronjs — APPROVED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (ultronjs) [reply .vscode/settings.json:3]: Otherwise good to go

**Comment by ShreyashMahajan** [comment]
> Looks good to merge.

thank you

File: .vscode/settings.json (hunk 1 above)
Comment (ShreyashMahajan) [reply .vscode/settings.json:3]: thank you, i will make the changes

**Pull request merged**